internal/configuration/configuration.go

```diff
@@ -131,18 +131,31 @@ func (m *Manager) Update(message models.DeviceConfigurationMessage) error {
 		return nil
 	}
 
-	log.Tracef("updating configuration: %v", message)
 	var errors error
+
+	newJson, err := json.Marshal(message)
+	if err != nil {
+		errors = multierror.Append(errors, fmt.Errorf("cannot marshal message to JSON: %s", err))
+		return errors
+	}
+
+	oldJson, err := json.Marshal(m.deviceConfiguration)
+	if err != nil {
+		errors = multierror.Append(errors, fmt.Errorf("cannot marshal message to JSON: %s", err))
```

Comment:
message is identical to previous one. pls add a notation that this is the existing/old configuration that failed (not sure if this can happen, but anyhow let's differentiate this from the previous message)

Reply:
fixed